Vagrantfile

```diff
@@ -9,6 +9,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.network :forwarded_port, guest: 8282, host: 8282
     config.vm.network :forwarded_port, guest: 80, host: 8000
     config.vm.network :forwarded_port, guest: 4443, host: 4443
+    config.vm.network :forwarded_port, guest: 777, host: 777
```

Comment:
ah damn, looks like it won't work, it doesn't like those in the 'location' section.

nginx: [emerg] "ssl_client_certificate" directive is not allowed here in /etc/nginx/sites-enabled/stage.uber.magfest.org.conf:53

there is something else that might work though I saw somewhere, setting ssl_verify_client to optional and then if in the jsonrpc location they don't have client-side verification, you 403 them